### PR TITLE
Update to fastify 4

### PR DESCRIPTION
**Pull request by kibertoad**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10, 12, 14, 16]
+        node-version: [14, 16]
```
Comment (jsumners) [inline .github/workflows/ci.yml:19]:
```suggestion
        node-version: [14, 16, 18]
```

**Review by jsumners — APPROVED** [review]
Looks good to me.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kibertoad) [reply .github/workflows/ci.yml:19]: Node 18 is exploding due to some issues which seem related to older Got, and newer got explodes due to migration to ESM, and addressing this is beyond this particular PR

Comment (jsumners) [reply .github/workflows/ci.yml:19]: I would love it if all Sindre modules were eradicated from any code base I am associated with.

**Pull request merged**